service/k8sclient/k8client.go

```diff
@@ -351,19 +349,15 @@ func (c *K8Client) getPerconaXtraDBClusters(ctx context.Context) ([]XtraDBCluste
 
 // getDeletingClusters returns clusters which are not fully deleted yet.
 func (c *K8Client) getDeletingClusters(ctx context.Context, managedBy string, runningClusters map[string]struct{}) ([]Cluster, error) {
-	var list meta.List
+	var list psmdb.PerconaServerMongoDBDeletingList
```

Comment:
it can be both type of pods

Reply:
Could you please explain what mean?
In k8s only one type of pod https://github.com/kubernetes/api/blob/master/core/v1/types.go#L3662-L3683

Reply:
list type is `psmdb.PerconaServerMongoDBDeletingList`, so it's better to use common type.

Reply:
`getDeletingClusters` can be called for both cluster types

Reply:
Renamed PerconaServerMongoDBDeletingList to PodList